client/lib/plugins/test/sanitize-section-content.js

```diff
@@ -36,6 +36,9 @@ test( 'should strip out content with non-whitelisted tags', () =>
 		'<p>alert("do bad things")👍</p>'
 	) );
 
+test( 'should strip out non-whitelisted children', () =>
```

Comment:
thanks!

Reply:
just pausing to reflect on how sad this would read in a will